course_content/notebooks/dask_intro.ipynb

```diff
@@ -1241,6 +1241,314 @@
+    "\n",
+    "Iris uses Dask to provide deferred data loading and operations :  Both cube data and coordinate values may be *either* Dask or Numpy arrays (i.e. [da.Array](http://dask.pydata.org/en/latest/array.html) or [np.ndarray](https://docs.scipy.org/doc/numpy/reference/generated/numpy.ndarray.html#numpy.ndarray)).\n",
+    "\n",
+    "Iris refers to Dask array data as \"lazy\" arrays, and provides a independent, non-Dask-specific API for handling them.\n",
+    "For example, the specific cube methods [Cube.lazy_data()](http://scitools.org.uk/iris/docs/v2.0/iris/iris/cube.html?highlight=lazy_data#iris.cube.Cube.lazy_data) and [Cube.has_lazy_data()](http://scitools.org.uk/iris/docs/v2.0/iris/iris/cube.html?highlight=has_lazy_data#iris.cube.Cube.has_lazy_data).  \n",
+    "\n",
+    "Iris operations that can work on lazy data to produce more lazy data (i.e. a deferred version of the operation) are referred to as \"lazy-preserving\".  These include, in particular :\n",
+    "\n",
+    "  * taking cube sub-sections, either with [indexing](http://scitools.org.uk/iris/docs/v2.0/userguide/subsetting_a_cube.html#cube-indexing) or [constraint extraction](http://scitools.org.uk/iris/docs/v2.0/userguide/subsetting_a_cube.html#cube-extraction).\n",
+    "  * cube [arithmetic operators](http://scitools.org.uk/iris/docs/v2.0/userguide/cube_maths.html#), ( + - \\* / \\** ).\n",
+    "  * statistical calculations with certain cube aggregators, such as [MEAN](http://scitools.org.uk/iris/docs/v2.0/iris/iris/analysis.html#iris.analysis.MEAN) and [STD_DEV](http://scitools.org.uk/iris/docs/v2.0/iris/iris/analysis.html#iris.analysis.STD_DEV)."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Example lazy operation : Wind calculations\n",
+    "\n",
+    "#### First grab some sample wind vector data\n",
+    "This is in cubes of U and V (i.e. eastward and northward winds)."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "filepath = iris.sample_data_path('wind_speed_lake_victoria.pp')\n",
+    "u_cube, v_cube = iris.load_cubes(filepath, ('x_wind', 'y_wind'))\n",
+    "\n",
+    "print 'U cube : \\n', u_cube\n",
+    "print\n",
+    "print 'V cube : \\n', v_cube.summary(shorten=True)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### These cubes have lazy data initially, which means they actually contain Dask arrays."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "print 'U cube : lazy =', u_cube.has_lazy_data()\n",
+    "print 'U cube : raw data content =', u_cube.core_data()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Iris in-built lazy operations : statistics\n",
+    "\n",
+    "Let's take average winds over the region.  \n",
```

Comment:
"Let's **calculate** average winds over the region." might be a more accurate statement.